docs/personas/personas.qmd

```diff
@@ -2,9 +2,9 @@
 title: Personas
 ---
 
- - **Infrastructure Engineer**: I create Azure infrastructure via Terraform code, and I am also known as the platform administrator or maintainer.
+ - **Infrastructure Engineer/Administrator**: I create Azure infrastructure via Terraform code, and I am also known as the platform administrator or maintainer.
  - **Data Scientist**: I writes data transformation code, and build models. I could also be an NHS analyst where the task is similar but I am likely to use different tools (SSMS, PowerBI etc. and rely less on inference).
- - **Internal Innovators**:
+ - **Internal Innovator/Developer**:
```

Comment:
I think the view is that an innovator is a user of flowEHR, not someone who develops it. What were you aiming for with this change?